src/current/_includes/releases/v23.2/v23.2.1.md

```diff
@@ -0,0 +1,161 @@
+
+<h3 id="v23-2-1-bug-fixes">Bug fixes</h3>
+
+- Fixed a durability bug in [Raft log](https://www.cockroachlabs.com/docs/v23.2/architecture/replication-layer#raft) storage, caused by incorrect syncing of filesystem metadata. It was possible to lose writes of a particular kind (`AddSSTable`) that are used by e.g. [`RESTORE`](https://www.cockroachlabs.com/docs/v23.2/restore). This loss was possible only under power-off or operating system crash conditions. Under such conditions, CockroachDB could enter a crash loop on node restart. In the worst case of a coordinated power-off/crash across multiple nodes this could lead to an unrecoverable loss of [Raft quorum](https://www.cockroachlabs.com/docs/v23.2/architecture/replication-layer#raft). [#115709][#115709]
+- Fixed a bug where large [jobs](https://www.cockroachlabs.com/docs/v23.2/show-jobs) running with execution locality (such as some [changefeeds](https://www.cockroachlabs.com/docs/v23.2/changefeeds-in-multi-region-deployments#run-a-changefeed-job-by-locality)) could result in the [gateway](https://www.cockroachlabs.com/docs/v23.2/architecture/life-of-a-distributed-transaction#gateway) node being assigned most of the work, causing performance degradation and cluster instability [#115388][#115388]
+- Fixed a bug that caused node crashes and panics when running [`INSERT`](https://www.cockroachlabs.com/docs/v23.2/insert) queries on [`REGIONAL BY ROW` tables with `UNIQUE` constraints or indexes](https://www.cockroachlabs.com/docs/v23.2/alter-table#add-a-unique-index-to-a-regional-by-row-table). The bug was only present in version 23.2.0-beta.1. [#115668][#115668]
+- Fixed a bug that existed only in 23.2 alpha and beta versions that could have caused side effects to happen out of order for [PL/pgSQL](https://www.cockroachlabs.com/docs/v23.2/plpgsql) routines in rare cases. [#115839][#115839]
+- Fixed a bug that existed since v23.1 that prevented naming [user-defined type (UDT)](https://www.cockroachlabs.com/docs/v23.2/create-type) parameters when dropping a [user-defined function](https://www.cockroachlabs.com/docs/v23.2/user-defined-functions) (or procedure). [cockroachdb/cockroach#115904][#115904]
+- Fixed a bug where [scheduled jobs](https://www.cockroachlabs.com/docs/v23.2/show-schedules) using [external storage providers](https://www.cockroachlabs.com/docs/v23.2/create-external-connection) could fail shortly after node startup. [#115693][#115693]
+- Locking tables (e.g., with [`SELECT FOR UPDATE`](https://www.cockroachlabs.com/docs/v23.2/select-for-update)) on the null-extended side of [outer joins](https://www.cockroachlabs.com/docs/v23.2/joins) (e.g., the right side of a `LEFT JOIN`) is now disallowed and returns an error. This improves compatibility with Postgres and prevents ambiguity in [locking semantics](https://www.cockroachlabs.com/docs/v23.2/architecture/transaction-layer#concurrency-control). This bug has existed since locking with `FOR UPDATE` was introduced. [#115878][#115878]
```

Comment:
my least favorite thing in our style guide :-} 

however I will comply ✅ 

from https://www.postgresql.org/docs/current/history.html : 

> Many people continue to refer to PostgreSQL as “Postgres” (now rarely in all capital letters) because of tradition or because it is easier to pronounce. This usage is widely accepted as a nickname or alias.